Include/nosVulkan/Device.h

```diff
@@ -294,7 +294,66 @@ struct nosVulkan_API Device : SharedFactory<Device>,
 
     Device(VkInstance Instance, VkPhysicalDevice PhysicalDevice, const nos::vk::Context* context);
     ~Device();
-    u64 GetLuid() const;
+
+    struct LUID
```

Comment:
This is a generic 16 bit id holder. No specific code for LUID. Uid16 or something like that.